### ISSUE TITLE
How should I test the trained model? 

### ISSUE THREAD

**Issue by QiaoJinming** [zhijiejia/ChangeNet-Pytorch#1]
I have successfully trained the model, but I don't know how to test the model,may i catch modified code，thank you very much

**Comment by zhijiejia** [comment]
Here is the test on the test dataset
![image](https://user-images.githubusercontent.com/43838913/114275849-b46f7380-9a56-11eb-9d30-7084483087e9.png)

**Comment by QiaoJinming** [comment]
> Here is the test on the test dataset
> ![image](https://user-images.githubusercontent.com/43838913/114275849-b46f7380-9a56-11eb-9d30-7084483087e9.png)

Thank you very much for your reply, IoU is satisfactory, but when I use two images (refer and test) to predict the mask (label), I get very bad results. According to your code, I use it like this "output"

`res = output.squeeze(0).cpu().numpy()
 plt.imshow(res)`

Got this result

![Cache_-7ad9432f28205081](https://user-images.githubusercontent.com/23067480/114276248-5fccf800-9a58-11eb-947c-30701f33c252.jpg)

The result is very frustrating, but the code you designed is very effective. I suspect that the method I used is wrong, so I want to ask you to get a copy of the execution code for generating the label from the refer and test images. If you interrupt, please understand.

Thanks again for your work

**Comment by QiaoJinming** [comment]
> Here is the test on the test dataset
> ![image](https://user-images.githubusercontent.com/43838913/114275849-b46f7380-9a56-11eb-9d30-7084483087e9.png)

Supplement the reply just now

Hyperparameters have not been modified
hardware
`nvidia RTX 2080`
software
`torch==1.7.0`
`PIL==8.0.1`
`python==3.7`

Thank you

**Comment by QiaoJinming** [comment]
Hello, I have replied to your answer, but I have not received a new notification. Do you have time to reply? The network architecture of this project is very influential. I sincerely look forward to your reply.

**Comment by zhijiejia** [comment]
All my code is here, and the effect of this network itself is not very good. I'm going to test the code again later。

**Comment by sssimpleboy** [comment]
> I have successfully trained the model, but I don't know how to test the model,may i catch modified code，thank you very much我已经成功训练了模型，但我不知道如何测试模型，我可以捕获修改后的代码，非常感谢

您好  请问您知道如何测试模型了吗？  我的ouput 是 2，224，224 我不知道怎么转化成 224，224  的黑白图 哈哈哈哈

**Comment by xiaoyangxiaoma** [comment]
您好，我想问一下数据集可以在哪里获取
